internal/controllers/gateway/gateway_controller.go

```diff
@@ -193,7 +193,7 @@ func (r *GatewayReconciler) gatewayClassMatchesController(obj client.Object) boo
 	if !ok {
 		r.Log.Error(
 			fmt.Errorf("unexpected object type"),
-			"gatewayclass watch predicate received unexpected object type",
+			"Gatewayclass watch predicate received unexpected object type",
 			"expected", "*gatewayapi.GatewayClass", "found", reflect.TypeOf(obj),
```

Comment:
https://github.com/Kong/kubernetes-ingress-controller/blob/5adc1813096c6871d81342e40ba76516b2f9f124/internal/controllers/gateway/gateway_controller.go#L210
is not updated.